### PR TITLE
Chessmasters for the J-Name Release

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette can you review. I won't be able to take this out of WIP until I post the POLL on talk and get that URL for this PR.

File: site/_posts/2019-07-29-vote-on-naming-j-release.html.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+author: jprause
+date: 2019-07-29 09:02:24 UTC
+tags: collaboration releases
+comments: true
+published: true
+---
+
+Vote for the J-Release!!! Here is the [Poll](http://talk.manageiq.org/t/name-of-j-release/????) on talk.manageiq.org
+
+While we close in on the Ivanchuk release, it’s time to think about what we'll name the J-Release.  In keeping with our nomenclature of using chess grandmasters and world champions as release names, here are the proposed names for the J-Release, as well some interesting background data:
```
Comment (cybette) [inline site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: as well as some interesting... (missing the second `as`)

File: site/_posts/2019-07-29-vote-on-naming-j-release.html.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- [Jansa, Vlastimil](https://en.wikipedia.org/wiki/Vlastimil_Jansa) Peak rating 2540 (July 1975)
+Jansa learned chess while in the hospital at the age of eight and at fourteen, became the youth champion of Prague. A deep and original thinker, Jansa has been one of the Czech Republic's leading players for many years, collecting experience across a range of chess activities.
+- [Johannessen, Leif Erlend](https://en.wikipedia.org/wiki/Leif_Erlend_Johannessen) Peak Rating 2564 (October 2005)
+Johannessen is a Norwegian chess player, and Norway's fifth grandmaster. He received his title in 2002, and was at the time Norway's second youngest grandmaster of all time. He picked up his first norm in Oslo, the second at Bermuda and finally the third in the Sigeman tournament in Malmö.
+- [Jones, Gawain](https://en.wikipedia.org/wiki/Gawain_Jones) Peak rating 2702 (May 2019)
+Jones began playing chess at the age of four, competing in his first tournaments at six. In early 1997 he hit the headlines and was featured on the front page of The Guardian newspaper when he became the youngest player in the world ever to beat an International Master in an official tournament game.
```
Comment (cybette) [inline site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: Perhaps meniton his nationality to be consistent with the others.

Comment (JPrause) [reply site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: Fixed these two items, as well as tried to make the verbiage clearer. (I hope)

File: site/_posts/2019-07-29-vote-on-naming-j-release.html.md [hunk 3]
```diff
@@ -9,18 +9,18 @@ published: true
...
 - [Johannessen, Leif Erlend](https://en.wikipedia.org/wiki/Leif_Erlend_Johannessen) Peak Rating 2564 (October 2005)
 Johannessen is a Norwegian chess player, and Norway's fifth grandmaster. He received his title in 2002, and was at the time Norway's second youngest grandmaster of all time. He picked up his first norm in Oslo, the second at Bermuda and finally the third in the Sigeman tournament in Malmö.
 - [Jones, Gawain](https://en.wikipedia.org/wiki/Gawain_Jones) Peak rating 2702 (May 2019)
-Jones began playing chess at the age of four, competing in his first tournaments at six. In early 1997 he hit the headlines and was featured on the front page of The Guardian newspaper when he became the youngest player in the world ever to beat an International Master in an official tournament game.
+Jones ia British chess player and began playing at the age of four, competing in his first tournaments at six. In early 1997 he hit the headlines and was featured on the front page of The Guardian newspaper when he became the youngest player in the world ever to beat an International Master in an official tournament game.
```
Comment (cybette) [inline site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: ia => is a

Comment (JPrause) [reply site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: Ugh, sorry for making you have to mine the text. But appreciate your :eyes:

Comment (cybette) [reply site/_posts/2019-07-29-vote-on-naming-j-release.html.md]: No worries, it's a 'talent' I've been blessed with, my 👀 tend to gravitate towards certain misbehaving texts. Or maybe it's a curse, as sometimes I'm too bothered by the typos/grammar nuances to pay enough attention to the actual content, which is what's important!

**Comment by cybette** [comment]
Will merge on Monday when you have added link to poll!

**Comment by JPrause** [comment]
@cybette I added the url for the poll. Ready for final review/merge. (can you also glance at the talk poll to verify the names look okay. One of the first names has two z's: Krzysztof)

**Pull request merged**